src/DDTrace/Integrations/Lumen/LumenIntegration.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace DDTrace\Integrations\Lumen;
+
+use DDTrace\Integrations\Integration;
+use DDTrace\Integrations\Lumen\V5\LumenIntegrationLoader;
+
+final class LumenIntegration extends Integration
+{
+    const NAME = 'lumen';
+
+    /**
+     * @var self
+     */
+    private static $instance;
+
+    /**
+     * @return self
+     */
+    public static function getInstance()
+    {
+        if (null === self::$instance) {
+            self::$instance = new self();
+        }
+        return self::$instance;
+    }
+
+    /**
+     * @return string The integration name.
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function requiresExplicitTraceAnalyticsEnabling()
+    {
+        return false;
+    }
+
+    /**
+     * Loads the integration.
+     *
+     * @return int
+     */
+    public static function load()
+    {
+        $instance = new self();
```

Comment:
Why not using the `self::getInstance()` method?

Reply:
Nice catch! Total copy/paste error. 🤦‍♂Fixed!

After everything else, I can submit a separate PR to make the `DDTrace\Integrations\Integration` constructor private, move `getInstance()` to the parent class and update this in other places so that we don't have this issue in the future. WDYT? :)